web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -63,6 +61,22 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
                 },
                 getMiniMapLinkTitle: function(values) {
                     return (this.isRecActive(values.uuid)) ? OpenLayers.i18n('collectionExistsMsg') : OpenLayers.i18n("addDataCollectionMsg");
+                },
+                getTitle: function(values) {
+                    var broader = [];  
+                    Ext.each(values.parameter, function(param) {
+                        var broaderTerms = this.classificationStore.getBroaderTerms(param, 2, 'Measured parameter');
+                        if(broaderTerms.length > 0) { 
+                            broader = broader.concat(broaderTerms); 
+                        }
+                        else {
+                            console.log("Could not find broader term for parameter '" + param  + "'");
```

Comment:
I think we should remove the console logs as they fire off a lot of responses on load... Maybe they should be error logged in the console instead if logging is required